### DISCUSSION TITLE
how to correctly configure automerge?

### DISCUSSION THREAD

**Discussion by 7opf** [renovatebot/renovate#9143] [Request Help] [answered]
When I configure a packageRule with automerge, I get MRs with automerge "disabled by config".

```json
  "packageRules": [
    {
      "matchUpdateTypes": ["patch", "pin", "digest"],
      "automerge": true,
      "gitLabAutomerge": true
    }
  ]
```

However, if I add automerge to the top level config, it works:

```json
  "gitLabAutomerge": true,
  "automerge": true,
  "packageRules": [
    {
      "matchUpdateTypes": ["patch", "pin", "digest"],
      "automerge": true,
      "gitLabAutomerge": true
    }
  ]
```
 
Is this expected behaviour? and if so, do I now need to explicitly disable automerge for other update types?

**Comment by rarkins** [comment, marked as answer]
Look up separateMinorPatch and set it to true
